controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/systemflags/v1/SystemFlagsDataArchive.java

```diff
@@ -206,4 +236,44 @@ public SystemFlagsDataArchive build() {
         }
 
     }
+
+    private static class JsonAccessor {
+        private final JsonNode jsonNode;
+
+        public JsonAccessor(JsonNode jsonNode) {
+            this.jsonNode = jsonNode;
+        }
+
+        public JsonAccessor get(String fieldName) {
+            if (jsonNode == null) {
+                return this;
+            } else {
+                return new JsonAccessor(jsonNode.get(fieldName));
+            }
+        }
+
+        public Optional<String> asString() {
+            return jsonNode != null && jsonNode.isTextual() ? Optional.of(jsonNode.textValue()) : Optional.empty();
+        }
+
+        public void forEachArrayElement(Consumer<JsonAccessor> consumer) {
+            if (jsonNode != null && jsonNode.isArray()) {
+                jsonNode.forEach(jsonNodeElement -> consumer.accept(new JsonAccessor(jsonNodeElement)));
+            }
+        }
+
+        public void removeCommentRecursively() {
```

Comment:
?

Reply:
PR on your way to remove this unused method